tests/testthat/test-data_storage-logfile.R

```diff
@@ -7,3 +7,67 @@ test_that("[LogFile] DataStorage should be able to insert and read", {
 
   test_common_data_storage(data_storage)
 })
+
+test_that("[LogFile] DataStorage should be able to insert and read events without details", {
+  log_file_path <- tempfile(fileext = ".txt")
+  withr::defer(file.remove(log_file_path))
+
+  data_storage <- DataStorageLogFile$new(log_file_path = log_file_path)
+  withr::defer(data_storage$close())
+
+  data_storage$insert(
+    app_name = "app_name",
+    type = "without_session"
+  ) %>% expect_silent()
+
+  data_storage$read_event_data() %>%
+    expect_silent() %>%
+    NROW() %>%
+    expect_equal(1)
```

Comment:
It is, I'm also removing the `expect_silent`, the reasoning is the same as in your other comment

We just cannot use the length, as with data.frames with 1 row, it will look at the number of columns